lib/pure/asyncfile.nim

```diff
@@ -466,6 +466,15 @@ proc write*(f: AsyncFile, data: string): Future[void] =
       addWrite(f.fd, cb)
   return retFuture
 
+proc setLen*(f: AsyncFile, len: int64) =
```

Comment:
Not that I can find, it seems you have to use something platform specific. I was planning on using ``AsyncFile`` for everything.